<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CertificateAssistant</class>
 <widget class="QDialog" name="CertificateAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="pages_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_namefields">
      <widget class="QWidget" name="formLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="namefields_layout">
        <item row="5" column="0">
         <widget class="QLabel" name="commonName_label">
          <property name="text">
           <string>Common Name</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="commonName_field"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="countryName_label">
          <property name="text">
           <string>Country Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="countryName_field">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="stateorprovince_label">
          <property name="text">
           <string>State/Province</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="stateorprovince_field"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="locality_label">
          <property name="text">
           <string>City (Locality)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="locality_field"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="organization_label">
          <property name="text">
           <string>Organization Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="organization_field"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="organizationalunit_label">
          <property name="text">
           <string>Organizational Unit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="organizationalunit_field"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="emailAddress_label">
          <property name="text">
           <string>Email Address</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="emailAddress_field"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_algorithm">
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="algorithm_layout">
        <item row="0" column="1">
         <widget class="QComboBox" name="algorithm_field">
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="algorithm_label">
          <property name="text">
           <string>Algorithm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="keysize_label">
          <property name="text">
           <string>Key Size</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="keysize_field"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_validity">
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="validity_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="validityperiod_label">
          <property name="text">
           <string>Validity (Period in Days)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="validityperiod_field">
          <property name="maximum">
           <number>16777215</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="validtiyperiod_from_label">
          <property name="text">
           <string>Valid from</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="validityperiod_from_field">
          <property name="text">
           <string>06/12/2018</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="validityperiod_until_label">
          <property name="text">
           <string>Valid until</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="validityperiod_until_field">
          <property name="text">
           <string>06/12/2020</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>CertificateAssistant</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>CertificateAssistant</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
